Formula/g/gcc.rb

```diff
@@ -5,27 +5,16 @@ class Gcc < Formula
   head "https://gcc.gnu.org/git/gcc.git", branch: "master"
 
   stable do
-    url "https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz"
-    mirror "https://ftpmirror.gnu.org/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz"
-    sha256 "e275e76442a6067341a27f04c5c6b83d8613144004c0413528863dc6b5c743da"
+    url "https://gcc.gnu.org/pub/gcc/snapshots/14.1.0-RC-20240503/gcc-14.1.0-RC-20240503.tar.xz"
+    version "14.1.0"
```

Comment:
Oh, right. Still RC.

Reply:
There are release snapshots on the mirrors already
http://ftp.ntua.gr/mirror/gnu/gcc/releases/gcc-14.1.0/